infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/SidecarToolingProvisioner.java

```diff
@@ -56,6 +57,7 @@ public SidecarToolingProvisioner(
     this.pluginBrokerManager = pluginBrokerManager;
   }
 
+  @Traced
```

Comment:
I think it would be useful to see traced `PluginBrokerManager#getTooling` method.

Reply:
:+1: